backend/clubs/views.py

```diff
@@ -2979,6 +2981,15 @@ def get_serializer_class(self):
         return UserProfileSerializer
 
 
+class ClubApplicationViewSet(viewsets.ModelViewSet):
+    permission_classes = [ProfilePermission | IsSuperuser]
```

Comment:
You probably don't want to use `ProfilePermission` in this case. Only officers should be able to create, edit, and delete club applications. A more appropriate permission class would be the one that the `Event` or `Testimonial` view sets use.